src/app/footer/footer.component.ts

```diff
@@ -7,6 +7,49 @@ import { Component, OnInit } from '@angular/core';
 })
 export class FooterComponent implements OnInit {
 
+  shopDepartment: string = "Shop departments";
+  shippingInfo: string = "Account & shipping info";
+  aboutUs: string = "About us";
```

Comment:
these strings will always be static. You can hardcode these in html file. For dynamic prop, we use this approach

Reply:
@akashsinha13, Those strings are hardcoded.